modules/luci-base/htdocs/luci-static/resources/network.js

```diff
@@ -3570,6 +3570,28 @@ WifiNetwork = baseclass.extend(/** @lends LuCI.network.WifiNetwork.prototype */
 		return ifname;
 	},
 
+	/**
+	 * Get the Linux VLAN network device names.
+	 *
+	 * @returns {string[]}
+	 * Returns the current Linux VLAN network device name as resolved
+	 * from `ubus` runtime information or empty array if this network
+	 * has no associated VLAN network devices.
+	 */
+	getVlanIfnames: function() {
+		var vlans = this.ubus('net', 'vlans'),
+		    ifnames = [];
+
+		if (vlans == null)
```

Comment:
... you can then subsequently drop this check and simply loop over the empty array below...